### ISSUE TITLE
apache2.service not found

### ISSUE THREAD

**Issue by elburro1887** [clonemeagain/osticket-vagrant#1]
The line to set the timezone in the Vagrantfile will fail upon with "apache2.service not found" when running "vagrant up" the first time, since it requires apache2, which gets installed afterwards via the provision.sh script.

so the timezone set should be run in the provision script or vagrantfile after installing apache2

**Comment by clonemeagain** [comment]
Thanks for the report mate.

**Comment by clonemeagain** [comment]
Yeah, that didn't fix it.. moving to provision script like you suggested, turns out Vagrant has script argument support, so I'll try that.
